gc/base/standard/Scavenger.hpp

```diff
@@ -624,10 +624,10 @@ class MM_Scavenger : public MM_Collector
 	/* methods used by either mutator or GC threads */
 	/**
 	 * All open copy caches (even if not full) are pushed onto scan queue. Unused memory is abondoned.
-	 * @param env Invoking thread. Could be master thread on behalf on mutator threads (threadEnvironment) for which copy caches are to be released, or could be mutator or GC thread itself.
-	 * @param threadEnvironment Thread for which copy caches are to be released. Could be either GC or mutator thread.
+	 * @param env Invoking thread, for which copy caches are to be released. Could be either GC or mutator thread.
+	 * @param final If true (typically at the end of a cycle), abandon TLH remainders, too. Otherwise keep them for possible future copy cache refresh.
 	 */
-	void threadFinalReleaseCaches(MM_EnvironmentBase *env);
+	void threadReleaseCaches(MM_EnvironmentBase *env, bool final = true);
```

Comment:
This change is good, however I am not a strong supporter of set parameters by default. For me would be better to explicitly add extra parameter at the caller side. You touch caller side any way so this would not involve wide spread change

Reply:
makes sense - will remove the default value.